README.md

```diff
@@ -215,6 +215,15 @@ Triton image as:
         ... # other files needed by mybackend
 ```
 
+Starting from 24.01, the default backend shared library name can be changed by
+providing the `runtime` setting in the model configuration. For example,
+
+```
+runtime: "my_backend_shared_library_name.so"
+```
+
+A model may choose a specific runtime implementation provided by the backend.
```

Comment:
Where would users find out about the python runtime implementation and how to find/use it?

Reply:
Oh sorry nevermind, this is the generic backend repo.